docs/processing-performance.asciidoc

```diff
@@ -5,40 +5,110 @@ APM Server performance depends on a number of factors: memory and CPU available,
 network latency, transaction sizes, workload patterns,
 agent and server settings, versions, and protocol.
 
-Let's look at a simple example that makes the following assumptions:
+// Assumptions
+// TBD
 
-* The load is generated in the same region as where APM Server and {es} are deployed.
-* We're using the default settings in cloud.
-* A small number of agents are reporting.
+// Method
+To help you understand how sizing APM Server impacts performance, we tested several scenarios:
 
-This leaves us with relevant variables like payload and instance sizes.
-See the table below for approximations.
-As a reminder, events are
+* Using the default hardware template on AWS, GCP and Azure on {ecloud}.
+* Comparing the performance when using OpenTelemetry (OTel) events on at least one of the service providers listed above (in this case, GCP).
```

Comment:
We have tested with Elastic apm agents and should keep otel out for now, until we have more advanced performance testing done.